README.md

```diff
@@ -1,17 +1,14 @@
 <div align="center">
   <img src="resources/mmdet-logo.png" width="600"/>
```

Comment:
In my opinion, this line is appropriate for `<h1>`, and many changes `##` -> `#` are unnecessary.
https://github.com/open-mmlab/mmdetection/commit/24e84170ee73caa15daff935d16cdcbd22ff3da8#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

E.g.
```html
<h1><img src="resources/mmdet-logo.png" alt="MMDetection" width="600"/></h1>
```